js/index.js

```diff
@@ -0,0 +1,35 @@
+let breadcrumbMap = {};
+
+const addPointTooBreadcrumbMap = function (point, link) {
+    breadcrumbMap[point] = link;
+}
+
+const breadcrumb = function (breadcrumbMap) {
+    document.getElementsByClassName("breadcrumb--wrapper")[0].classList.remove("breadcrumb--none");
+    const d = document.getElementsByClassName("breadcrumb")[0]
+    const c = document.createElement("span")
+    c.innerHTML = "»";
+    let a = document.createElement("a")
+    a.innerHTML = "Main";
+
+    a.href = "/"
+    d.appendChild(a);
+    d.appendChild(c);
+    for (key in breadcrumbMap) {
+        if (breadcrumbMap.hasOwnProperty(key)) {
+            if (!breadcrumbMap[key]) {
+                a = document.createElement("span")
+                a.innerHTML = key;
+                d.appendChild(a);
+            } else {
+                a = document.createElement("a")
+                const c = document.createElement("span")
+                c.innerHTML = "»";
```

Comment:
did you check it in IE?
maybe it's better to use unicode characters
but if in all browsers it works fine - let it be